sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1285,10 +1285,13 @@ def test_leading_terms():
     assert sin(S.Half).as_leading_term(x) == sin(S.Half)
     assert cos(1/x).as_leading_term(x) == AccumBounds(-1, 1)
     assert cos(S.Half).as_leading_term(x) == cos(S.Half)
+    assert sec(1/x).as_leading_term(x) == AccumBounds(S.NegativeInfinity, S.Infinity)
+    assert csc(1/x).as_leading_term(x) == AccumBounds(S.NegativeInfinity, S.Infinity)
+    assert tan(1/x).as_leading_term(x) == AccumBounds(S.NegativeInfinity, S.Infinity)
+    assert cot(1/x).as_leading_term(x) == AccumBounds(S.NegativeInfinity, S.Infinity)
 
     for func in [tan, cot]:
-        for a in (1/x, S.Half):
-            eq = func(a)
+            eq = func(S.Half)
```

Comment:
This should be re-indented to 4 spaces. Also, can `sin` and `cos` be included in the loop as well?